### PR TITLE
Created landing page

### LINKED ISSUE #52: Add the landing page to the app
This is Mairead's sketch of the landing page.

image

The idea to my understanding would be for there to be some info under the heading "SeroTracker" and then a button that redirects to "SarsCov2Tracker" as well as arbotracker (that's what those two buttons are)

### PR DESCRIPTION
**Pull request by SeanKennyNF**
Resolves #52 

Relevant discussion thread: https://3.basecamp.com/5061858/buckets/30589330/messages/6817918424

NewLandingPageScreenshot3

### CODE REVIEW

File: app/homepage-tile.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.homepage-tile {
```
[the remark this answers is not in the document]
Comment (SeanKennyNF) [reply app/homepage-tile.css:1]: https://github.com/PathoTracker/Pathotracker/issues/60 :+1:

File: app/homepage-tile.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        maxWidth: backgroundImage.width,
+        maxHeight: backgroundImage.height,
+      }}
+    >
+      <div className={"homepage-tile-text-container"}>
+        <h2 className={"homepage-tile-header"}> {header} </h2>
+        <p> {subtitle} </p>
+        <button
+          className={"homepage-tile-button"}
+          onClick={() => router.push(route)}
```
[the remark this answers is not in the document]
Comment (SeanKennyNF) [reply app/homepage-tile.tsx:35]: Sorry man, I'd have to do a bit more research here, a bit unqualified to answer this at the moment. A quick search just gave me some pretty unhelpful conflicting answers so I'm not confident in them lol

[the remark this answers is not in the document]
Comment (HimanshuRanka) [reply app/homepage-tile.tsx:39]: Ok sounds good ahahaha was just curious we can look into it more

**Review by HimanshuRanka — APPROVED** [review]
Looks great to me! Really like the homepage :)

**Pull request merged**